roles/letsencrypt/tasks/dns-challenge.yml

```diff
@@ -55,3 +55,9 @@
   when:
     - letsencrypt_do_dns_challenge | bool
     - letsencrypt_dns_provider == "pebble"
+
+- name: do dns challenge openstack
+  include_tasks: dns-challenge-os.yml
```

Comment:
`os` is ambigous here, it could mean `operating system`. Can you rename it to `openstack`, please? Same goes for all other variables with `os` in it.

Reply:
3e171f5